extensions/cache/runtime/src/main/java/io/quarkus/cache/augmented/interceptor/CacheKeyBuilder.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.cache.augmented.interceptor;
+
+import java.util.Arrays;
+
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * @author Gwenneg Lepage
+ */
+@ApplicationScoped
+public class CacheKeyBuilder {
```

Comment:
The key used to access an entry from a cache is composed of one to several elements which come from `@CacheKey`-annotated method parameters.

Reply:
No longer relevant: @CacheKey annotations are not mandatory anymore on method parameters.